README.md

```diff
@@ -56,85 +48,39 @@ This plugin use the new [miio](https://github.com/aholstenson/miio) version 0.15
       "id": 17,
       "name": "Kitchen"
     }
-  ],
-  "zones": [
- {
-      "name":"Family Room (x2)",
-      "zone":[[25000,25000,32000,32000,2]]
-    },
-    {
-      "name":"Bedroom",
-      "zone":[[21000,32000,24000,37000,1]]
-    },
-    {
-      "name":"Bedroom & Family Room",
-      "zone":[ [21000,32000,24000,37000,1],  [25000,25000,32000,32000,1]]
-    }
   ]
- }
-],
-```
-
-- Example `config.json` with two vacuums:
-
-```
-"accessories": [
- {
-  "accessory": "XiaomiRoborockVacuum",
-  "name": "Xiaomi Mi Robot Vaccum 1st Generation",
-  "ip": "192.168.1.150",
-  "token": "abcdef1234567890abcdef1234567890",
-  "pause": false,
-  "dock": true,
-  "waterBox": false
  },
- {
-  "accessory": "XiaomiRoborockVacuum",
-  "name": "Xiaomi Roborock S50 Vaccum 2nd Generation",
-  "ip": "192.168.1.151",
-  "token": "1234567890abcdef1234567890abcdef",
-  "pause": false,
-  "dock": true,
-  "waterBox": false
- }
+ "gotoTarget": {
+    "name": "Go to Bin",
+    "target": [
+          21659,
+          33063
+    ]
+  }
```

Comment:
Actual change to include in a PR

Reply:
> @maisun thank you for this PR. It really highlights some nice additions we can do this module. I really appreciate your time on doing this and all the hours of developing and testing you must have gone through to get this working for you.
> 
> I'm afraid I can't merge this PR as is right now because it will actually remove some features that have been added over time. Like the autoroom generation improvements or the multi-zone cleaning support, only to mention a few.
> 
> I've highlighted what looks to me like actual changes that can be implemented on top of the current `next` branch.
> 
> If you can update this PR (or create another one) with only those changes, that'd be amazing. If not, I think I've got some time on Friday to work on this module and cherry-pick your changes to bring them into this module.
> 
> Thank you again!

Hi, I highlighted all changes by me with a comment "changed by Mai" so it should be possible to cherry pick what's been added. I will write my intentions below. Thank you